Einthusan/presence.ts

```diff
@@ -0,0 +1,194 @@
+var presence = new Presence({
+    clientId: "702375041320484944"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+var search: string, title: string, director: string, video: any, timestamps: any[];
+let Name: any;
+
+function getTimestamps(videoTime: number, videoDuration: number): any {
+    var startTime = Date.now();
+    var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+    return [Math.floor(startTime / 1000), endTime];
+}
+function getSeconds(videoTime: string, videoDuration: string): any{
+    var a = videoTime.split(':');
+    var b = videoDuration.split(':');
+    var secondsStart = (+a[0]) * 60 * 60 + (+a[1]) * 60 + (+a[2]);
+    var secondsEnd = (+b[0]) * 60 * 60 + (+b[1]) * 60 + (+b[2]);
+    return timestamps = getTimestamps(Math.floor(secondsStart), Math.floor(secondsEnd));
+}
+
+presence.on("UpdateData", async() => {
+    var presenceData = {
+        largeImageKey: "logo",
+        details: "",
+        startTimestamp: 0,
+        smallImageKey: "",
+        smallImageText: "",
+        state: "",
+        endTimestamp: 0
```

Comment:
remove all, just only set the logo here

Reply:
If I do that I get 74 errors.
![image](https://user-images.githubusercontent.com/25113575/80611090-01e20480-89ef-11ea-88d1-1f1d43fae5c2.png)

Reply:
are you not editting this in your local repo (so you have the index files aswell)

Reply:
This is my local repo. The errors revolve around details, startTimestamp, smallImageKey, smallImageText, state, and endTimestamp need to be defined.
![image](https://user-images.githubusercontent.com/25113575/80612312-a7e23e80-89f0-11ea-9180-a4efa5b19c61.png)

Reply:
I do edit in my repo, but for some reason if I don't have the above values trypescript goes crazy.
Javascript does however work fine.
![image](https://user-images.githubusercontent.com/25113575/80612591-03acc780-89f1-11ea-8f19-95a3cc3fd534.png)

Reply:
you shouldnt get that issue if you have the index.d.ts files 
![image](https://user-images.githubusercontent.com/34792838/80613864-11d50500-8a3e-11ea-9439-466f9807aae3.png)

Reply:
![image](https://user-images.githubusercontent.com/25113575/80614206-04465d80-89f3-11ea-9683-c235cd1e63ef.png)

Reply:
nvm I know why:
var presenceData = {
        largeImageKey: "logo"
    };
should be:
var presenceData: presenceData = {
        largeImageKey: "logo"
    };

Reply:
Found the fix
![image](https://user-images.githubusercontent.com/25113575/80614841-cb5ab880-89f3-11ea-86bf-a9c9c8dcdca1.png)